Makefile.kokkos

```diff
@@ -972,12 +976,17 @@ ifeq ($(KOKKOS_INTERNAL_USE_CUDA_ARCH), 1)
 endif
 
 ifeq ($(KOKKOS_INTERNAL_USE_OPENMPTARGET), 1)
+	# Not necessary for the Cray compiler.
+  ifeq ($(KOKKOS_INTERNAL_COMPILER_CRAY_CLANG), 0)
   ifeq ($(KOKKOS_INTERNAL_COMPILER_CLANG), 1)
     KOKKOS_INTERNAL_CUDA_ARCH_FLAG=-fopenmp-targets=nvptx64 -Xopenmp-target -march
   endif
+  endif
   KOKKOS_INTERNAL_USE_CUDA_ARCH = 1
 endif
 
+# Do not add this flag it its the cray compiler.
+ifeq ($(KOKKOS_INTERNAL_COMPILER_CRAY_CLANG), 0)
```

Comment:
Please also indent accordingly.
So the Cray compiler doesn't take any architecture flags at all?!
Does this also apply if compiling for the `Cuda` backend?

Reply:
No Cray compiler does not take any architecture flags. I was also surprised by that update. Why would we compile `Cuda` backend with Cray compiler, do you mean using Cray Wrappers? `Cuda` backend points to `nvcc_wrapper` anyways right?